apps/wallet-ui/src/app/[ecosystem]/login/auth.ts

```diff
@@ -0,0 +1,68 @@
+"use server";
```

Comment:
to make it safer also add `import "server-only"`; so it will crash if it ever runs on client